src/Sylius/Component/Payment/Model/PaymentRequest.php

```diff
@@ -41,7 +41,7 @@ public function __construct(
 
     public function getId(): ?string
     {
-        return $this->getHash()?->toBinary();
+        return $this->getHash()?->__toString();
```

Comment:
```suggestion
        return $this->getHash()?->toString();
```
Pretty much the same thing, but this one's a bit less magic

Reply:
@NoResponseMate is there a reason to not use `\Stringable` method?

Reply:
Not really, it just looks better 🤷

Reply:
Oh, nevermind then, apparently it does not exist in Symfony 6.4

Reply:
Ok I revert this ;)